src/main/java/org/stellar/sdk/CreatePassiveSellOfferOperation.java

```diff
@@ -109,13 +105,25 @@ public Builder(
         @NonNull Asset selling,
         @NonNull Asset buying,
         @NonNull String amount,
-        @NonNull String price) {
+        @NonNull Price price) {
       this.selling = selling;
       this.buying = buying;
       this.amount = amount;
       this.price = price;
     }
 
+    /** An alias for {@link Builder#Builder(Asset, Asset, String, Price)} */
+    public Builder(
+        @NonNull Asset selling,
+        @NonNull Asset buying,
+        @NonNull String amount,
+        @NonNull String price) {
+      this.selling = selling;
```

Comment:
for consideration, convert and delegate to the other method rather than implement locally:
```
builder(selling, buying, amount, Price.fromString(price))
```